examples/existing-resources/README.md

```diff
@@ -1,8 +1,8 @@
-# COS Bucket without encryption using an existing COS instance and Key Protect instance + Keys
+# COS Buckets without encryption using an existing COS instance and Key Protect instance + Keys
 
 An end-to-end example that will:
 - Create a new resource group (if existing one is not passed in).
 - Create a new Key Protect instance, Key Ring, and Key in the given resource group and region.
 - Using the terraform-ibm-cos module, create a new Cloud Object Storage instance in the given resource group and region (with no buckets).
 - Create an IAM Access Policy to allow Key Protect to access COS instance (outside of the terraform-ibm-cos module).
-- Using the terraform-ibm-cos module, create a COS Bucket without encryption using the existing COS instance, Key Protect instance + Keys created at the start of this example.
+- Using the terraform-ibm-cos module, create a COS Buckets without encryption using the existing COS instance, Key Protect instance + Keys created at the start of this example.
```

Comment:
replace `COS Bucket` with Cloud Object Storage bucket (lowercase b and full name).

Can you combine this with the list item that identifies that you're creating a COS instance? Is there a reason to separate them? 

For example, 
Suggestion
```
- A Cloud Object Storage instance in the resource group and region with buckets that are not encrypted.
```